tests/foreman/cli/test_abrt.py

```diff
@@ -76,7 +76,7 @@ def test_abrt_4(self):
 
         """
 
-    def test_abrt_5(self):
+    def test_positive_cli_report(self):
```

Comment:
What is the difference between this report generating method and the others? They're all using the CLI so this name doesn't really tell me what is going on. Should this be a `test_positive_search_report_by_id` or something similar?